service/git/scanner.go

```diff
@@ -217,6 +217,10 @@ func NewChangeFilterScanner(scanner lookout.ChangeScanner, include, exclude stri
 	}
 
 	fn := func(ch *lookout.Change) (bool, error) {
+		if ch.Head == nil {
```

Comment:
Can you please explain when this happen?

Reply:
When you delete a file in git.